deploy/cht_sync/values.yaml.template

```diff
@@ -0,0 +1,25 @@
+
+postgres:
+  enabled: true  # if true a postgres service will not be created in this cluster
+  user: "postgres"
+  password: "postgres"
+  db: "data"
+  schema: "v1"
+  table: "medic"  
+  host: "postgres" # if postgres is outside the cluster
+  port: 5432 # if postgres is outside the cluster
+
+cht_pipeline_branch_url: "https://github.com/medic/cht-pipeline.git#main"
+
+couchdb:
+  user: "medic"
+  password: "password"
+  dbs: "medic" # space separated list of databases you want to sync e.g "medic medic_sentinel"
+  host: "couchdb"
+  port: "5984"
```

Comment:
So this is going to run within the same k8s network as Couch? Otherwise, Couch is not accessible and all requests need to pass through API

Reply:
yea couchdb would normally not be in the same network and host would be the public API url. this example value is confusing